### ISSUE TITLE
Dependency issue

### ISSUE THREAD

**Issue by umarmw** [jflheureux/jss-app-templates#1]
Hello,

It seems the tutorial is out of date. I tried to execute it directly on the "react-helix" folder but got this error:

```
Failed to compile

./node_modules/history/esm/history.js
Module not found: Can't resolve '@babel/runtime/helpers/esm/extends' in 'C:\inetpub\wwwroot\Sitecore\Experimental\poc\jss-app-templates\react-helix\node_modules\history\esm'
```

Any idea?

**Comment by jflheureux** [comment]
Indeed, this repo was created with JSS Technical Preview 4 (TP4). You could compare the react-helix folder with the latest official React sample ( https://github.com/Sitecore/jss/tree/master/samples/react ) and merge what needs to be merged. Then you can create a JSS app from a template stored in a local folder of your hard drive with a special command line argument.

**Comment by ksbhatt1107** [comment]
I resolved this issue by commenting the module reference in history.js file at two places -
1) node_modules\react-router\node_modules\history\esm\history.js
2) node_modules\react-router-dom\node_modules\history\esm\history.js

Commented first line - "import _extends from '@babel/runtime/helpers/esm/extends';" in above files. 

My app started working.
